eng/common/README.md

```diff
@@ -4,9 +4,9 @@ The `eng/common` directory contains engineering files that are common across the
 It should remain relatively small and only contain textual based files like scripts, configs, or templates. It
 should not contain binary files as they don't play well with git.
 
-# Updating
```

Comment:
Note to self: Once merged, open a PR in azure-sdk-tools with the changes to eng/common.

Reply:
What caused this to be a H2 instead of H1?

Reply:
[MD025](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md025) - Multiple top level headings in the same document

Reply:
Thanks. Would you mind also filing another issue in azure-sdk repo to consider using this MD linter for all the language repos?

Reply:
> Note to self: Once merged, open a PR in azure-sdk-tools with the changes to eng/common.

Done in https://github.com/Azure/azure-sdk-tools/pull/892 by @chidozieononiwu. Thanks!

Reply:
> Thanks. Would you mind also filing another issue in azure-sdk repo to consider using this MD linter for all the language repos?

Done. See https://github.com/Azure/azure-sdk/issues/1698